code/modules/reagents/newchem/food.dm

```diff
@@ -22,8 +22,10 @@ datum/reagent/egg
 
 datum/reagent/egg/on_mob_life(var/mob/living/M as mob)
 	if(!M) M = holder.my_atom
-	if(prob(5))
+	if(prob(8))
 		M.emote("fart")
+	if(prob(3))
+		M.reagents.add_reagent("cholesterol", rand(1,2))
```

Comment:
This should also scale on volume in some manner, probably on the probability so that it's not so swingy